src/main/java/com/sylver/watchwithme/model/ClientEvent.java

```diff
@@ -0,0 +1,49 @@
+package com.sylver.watchwithme.model;
+
+import java.util.OptionalInt;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+/**
+ * This class is the form all events coming from clients will take.
+ */
+public class ClientEvent {
+
+	public enum Type {
+		PLAY,
+		PAUSE,
+		BUFFERING_START,
+		BUFFERING_COMPLETE,
+		SEEK
+	}
```

Comment:
Why do you keep the enum like this? Does the enum now belong to the ClientEvent class?